install.sh

```diff
@@ -113,18 +115,15 @@ function install_extension() {
 }
 
 function override_version() {
-    cat .mvn/extensions.xml | awk -v groupId="org.arquillian.smart.testing" -v version=$1 -v RS="</extension>" '
-      $0 ~ "<groupId>" groupId "</groupId>" {
-        sub("<version>.*</version>", "<version>" version "</version>")
-      }
-      {printf "%s", $0 RS}' | sed '$ d' > .mvn/extensions-new.xml
+    sed -e "s/<version>.*<\/version>/<version>$1<\/version>/g" .mvn/extensions.xml > .mvn/extensions-new.xml
```

Comment:
What if you have more than one extension registered already? This won't work in such a case.

Reply:
We have to fix this use case prior to merge